regoservice/eval.go

```diff
@@ -67,10 +68,15 @@ func (regoEvaluator *regoEvaluator) Eval(in map[string]interface{}, serverUrl st
 		return nil, err
 	}
 
+	image_url_part, err := buildImageUrlPart(data)
```

Comment:
it's better to build url in rego. As there are could be other custom template implementation which may require bare server url

Reply:
Done